pyproject.toml

```diff
@@ -40,6 +41,9 @@ Documentation = "https://github.com/mrc-ide/outpack#readme"
 Issues = "https://github.com/mrc-ide/outpack/issues"
 Source = "https://github.com/mrc-ide/outpack"
 
+[project.scripts]
+orderly = "orderly.cli:cli"
```

Comment:
Possibly controversial comment here: if this is `orderly` then we have the situation where the python version of this and an R version can't really be installed side-by-side (we had an R based cli in orderly1 based on docopt; this has not been copied over to orderly2 yet, and might not).  I guess this loops us back to the issue with a good name for this package that stands alone but references that it's related to orderly

Reply:
Yeah definitely something I want to change eventually. We also can't use `outpack`, because that's what the Rust code uses :disappointed:

I think I've come to think we should go with `pyorderly`, both as the CLI tool and as the PyPI name and import name (and maybe repo). Avoids any accidental overwriting with the existing orderly on PyPI and is descriptive albeit a bit unimaginative.

I'd prefer to merge this as is and rename everything in one go though.

Reply:
One day we'll have a single unified CLI that can run both R and Python reports and we can call that one `orderly`